lib/web_catalog/object_graph_importer.es6.js

```diff
@@ -134,31 +134,59 @@ foam.CLASS({
           if (stat.isFile()) {
```

Comment:
Does the pattern on 128 still make sense?

Reply:
No. Good point.

Reply:
I would have gone further and said that the name format doesn't matter at all.

Reply:
The trouble with that is that if anything else happens to live in that directory, we attempt to interpret it as valid JSON. I've run into that problem before.